.changelog/9ebe24c4791541e0840da49eab6f9d97.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "9ebe24c4-7915-41e0-840d-a49eab6f9d97",
+    "type": "feature",
+    "description": "add client cfg to opt-in/out checksum behavior and change its default algorithm",
```

Comment:
1. This needs to be a complete and correct sentence.
2. Frankly I think we're going to have to give a lot more detail here. This is an implicit behavioral change. We can discuss externally.

Reply:
Make sense after seeing the accountid changelog pr